core/mondoo-linux-security.mql.yaml

```diff
@@ -1185,13 +1174,16 @@ queries:
         ```
   - uid: mondoo-linux-security-changes-to-system-administration-scope-sudoers-is-collected
     title: Ensure changes to system administration scope (sudoers) is collected
-    impact: 80
+    impact: 50
+    props:
+      - uid: auditFiles
+        title: Return the content from all /etc/audit/rules.d and /etc/audit/audit.rules
+        mql: |
+          AuditFiles = files.find(from: "/etc/audit/rules.d",regex:'.*\.rules$' , type: "file").list.map(path) + ["/etc/audit/audit.rules"]
```

Comment:
nit: from readability I find snakeCase with lower case at the beginning easier to read and would be more consistent with the rest of the policy

Reply:
yes, you are right

Reply:
done